README.md

```diff
@@ -291,6 +291,27 @@ if err != nil {
 fmt.Print(status, status.ChannelId)
 ```
 
+## Local web-based godoc
```

Comment:
I am not sure about including this in the README, which is really documentation for people who are using the library rather than people developing the library. If we include a "Local web-based godoc" section, 
we should at least add a sentence on why anyone would want to use this, and in what way this is different from any other Go library.

Reply:
- If developers want to update documentation for `ably-go`, say for a `method` or `class`, they need a way to `preview` it locally, without pushing to `CI` first, so that it's easy to make a quick correction, rather than waiting for CI to update it.
- For me personally, I had to look around for tools to generate the doc, most of the time answers were confusing or not available since the community is small. It's better if we have these standard tools documented somewhere.
- Also, I strongly believe this is a must setup while working on a new feature/refactoring where it will be needed to add documentation for the code, developer can have a quick preview and can correct it based on the presentation.

Reply:
OK, so I would say, lets put this under a new section called "Contributing" or something similar,
to make it clear that this is aimed at the small minority of people who are _contributing to the library itself_, rather 
than _using_ the library to add realtime abilities to their own system.

Reply:
This should be moved to [CONTRIBUTING.md](https://github.com/ably/ably-go/blob/main/CONTRIBUTING.md)

Reply:
Moved as per suggestion 👍